server/app_multi/app.py

```diff
@@ -0,0 +1,163 @@
+import os
+import datetime
+
+from flask import Flask, redirect, current_app, make_response, render_template
+from flask import Blueprint, request
+from flask_caching import Cache
+from flask_compress import Compress
+from flask_cors import CORS
+from flask_restful import Api, Resource
+
+from server.data_common.utils import Float32JSONEncoder, MatrixDataLoader
+
+from http import HTTPStatus
+from functools import wraps
+import server.common.rest as common_rest
+from server.common.utils import path_join
+
+web_bp = Blueprint("webapp", "server.common.web", template_folder="templates")
+
+
+@web_bp.route("/")
+def index():
+    # FIXME with a splash screen that includes a listing of all the datasets.
+    # or perhaps a login screen if this is a hosted environment
```

Comment:
or better yet, a config-driven redirect?

Reply:
Yes, this needs some design work.  I'll punt it to a followup release, if that's okay with you.